### ISSUE TITLE
customisable "pvp overlay" 

### ISSUE THREAD

**Issue by Doogie13** [kami-blue/client#1923]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
Something similar to a pvp hud (shows if a module is enabled but also if it is disabled) which would be customisable with own module list. you should also be able to set custom names for the modules (eg CrystalAura could display as CA)

**Describe alternatives you've considered**
other clients pvp hud

**Additional context**
![image](https://user-images.githubusercontent.com/62295632/107207555-54c5ff00-69f8-11eb-8fb3-898decabdd9f.png)

**Comment by l1ving** [comment]
I've already said no to this in the past, such as in #1563 and #620. It would also be blocked by #1068, for list setting support. 

Can you give some reasoning as to why this would be a *useful* feature?

**Comment by Doogie13** [comment]
> Can you give some reasoning as to why this would be a useful feature?

It would be more useful than the array list as you can easier keep track of things like CA in PVP and abbreviate things to take up less room.

**Comment by l1ving** [comment]
:thinking: my module list is only like 5 long and I have 40 other modules usually enabled

**Comment by Doogie13** [comment]
ig. it would be nice even to have a display name for modules so you can abreviate module names on the arraylist tho (like phobos i think)

**Comment by nopjmp** [comment]
instead of a text overlay in one corner, would a WeakAuras system be better?

I've seen the text overlay done and it seems out of the way of the actions for me to remember anything.

<img width="585" alt="image" src="https://user-images.githubusercontent.com/5185561/107238918-f85de200-69ed-11eb-9813-03c9d77dd0d1.png">

**Comment by l1ving** [comment]
Like, icons for some combat modules as an hud?

**Comment by nopjmp** [comment]
> Like, icons for some combat modules as an hud?

Yes also possibly moving or adding new icons for potion effects and absorption and such.

Maybe even tick timers for various combat modules that are happening to let you know when something is delayed/on. I've not see a client really do something like this yet.

**Comment by l1ving** [comment]
That would be much more useful than a module list that has a renaming option, yea
